pdns/recursordist/mtasker_fcontext.cc

```diff
@@ -132,6 +141,7 @@ extern "C"
                   static_cast<fcontext_t>(args->prev_ctx), 0);
 #else
     transfer_t res = jump_fcontext(t.fctx, 0);
+    CET_ENDBR;
```

Comment:
Do you think it would be possible to contribute this upstream, or does it have to live in the code resumed from `jump_fcontext`?

Reply:
I think the latter.

Reply:
Alright :+1: